.github/workflows/image.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: checkout code
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
        with:
          # Fetch all history for all tags and branches because we need tags for building the version information.
          fetch-depth: 0
```

Reply:
Suggest adding a comment on it.